apps/salesforce-commerce-cloud/src/components/dialog/CategorySearchResults.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { css } from '@emotion/react';
+
+import tokens from '@contentful/f36-tokens';
+import { Box, Card, Flex, Text } from '@contentful/f36-components';
+import { TagsIcon } from '@contentful/f36-icons';
+
+import {
+  SearchResultsProps,
+  SearchResultProps,
+  height,
+  stickyHeaderBreakpoint,
+  headerHeight,
+} from './SearchPicker';
+
+const CategorySearchResults = (props: SearchResultsProps) => {
+  const resultsWrapperStyle = css`
+    max-height: ${height}px;
+    padding: ${tokens.spacingL};
+    padding-bottom: 0;
+    @media screen and (min-height: ${stickyHeaderBreakpoint}px;) {
+      padding-top: ${tokens.spacingL + headerHeight}px;
+    }
+  `;
+
+  return (
+    <Box as="section" css={resultsWrapperStyle}>
+      {props.searchResults.map((result) => (
+        <CategorySearchResult
+          key={`${result.catalogId}:${result.id}`}
+          fieldType={props.fieldType}
+          onItemSelect={props.onItemSelect}
+          selected={props.selectedItems}
+          result={result}
+        />
+      ))}
+    </Box>
+  );
+};
+
+const CategorySearchResult = (props: SearchResultProps) => {
```

Comment:
`CategorySearchResult` could be broken out into it's own file just to keep this file a bit more focused